eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -49,41 +67,15 @@ jobs:
           Artifacts: ${{parameters.Artifacts}}
           DocGenerationDir: "$(Build.SourcesDirectory)/doc/ApiDocGeneration"
           LibType: 'client'
-  - job: "Analyze"
```

Comment:
I'm fine with you splitting out the code generation and making it run in parallel but I'd like to keep the other stuff in the Analyze step for consistency across languages and to provide a place to run static analysis type validation. Currently it doesn't look like it is even downloading the packages but it should be for the Component Governance check. @chidozieononiwu can you please take a look at that?

Reply:
Do you want me to revert it?

Reply:
> run static analysis type validation

In addition, we run most of that (roslyn analyzers) during build

Reply:
Yes please revert back to keeping the Analyze job, I know there are at least some things like Component Governance reporting that only looks at our Analyze jobs for reporting purposes. 

As for static analyzers I'm not necessarily talking about the code analyzers that would be run as part of compiling. I'm more meaning other policies like readme/link checks/component governance/ (possibly policheck and credscan in the future).